src/common/restext.cpp

```diff
@@ -89,6 +89,9 @@ void InitializeRestext()
     stringsText[RT_SETUP_KEY1]       = TR("1) First click on the key you want to redefine.");
     stringsText[RT_SETUP_KEY2]       = TR("2) Then press the key you want to use instead.");
 
+    stringsText[RT_MODS_UNLOADED]     = TR("Unloaded Mods:");
```

Comment:
After modifying restext.cpp you should always run `make update-pot` to update translation files (I keep promising myself that I'll add a comment describing that but I always forget)

Reply:
Oh... Okay...